catalogue/webapp/package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "dev": "nodemon --watch server.js server.js",
     "build": "next build",
+    "deploy_bundle_analysis": "aws s3 sync .dist s3://dash.wellcomecollection.org/bundles  --only-show-errors --acl public-read",
```

Comment:
nit: standard on naming is `camelCase`

Reply:
https://github.com/wellcomecollection/wellcomecollection.org/pull/5534